client/src/containers/profile/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { Divider, 
+          Avatar,
+          RaisedButton 
+        } from 'material-ui'
+import { push } from 'react-router-redux'
+
+import './style.css'
+
+const Profile = props => {
+	const {
+		user: {
+			firstName,
+			lastName,
+			experience,
+      linkedIn,
+      email
+		}, changePage
+	} = props
+
+  return (
+		<div>
+			<h1 className='title'>{firstName} {lastName}</h1>
+      <div className='title'><Avatar size={60} src="./blank-portrait.png" /></div>
```

Comment:
For a blank profile portrait we should use a consistent image, material ui has a person icon that you can just import and modify the size. I would suggest using that one.